include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -8012,12 +7626,80 @@ FEFaceEvaluation<dim,
 
   this->cell_type = this->matrix_info->get_mapping_info().cell_type[cell_index];
   this->cell      = cell_index;
-  this->face_orientation = 0;
-  this->subface_index    = GeometryInfo<dim>::max_children_per_cell;
-  this->face_no          = face_number;
+  this->subface_index = GeometryInfo<dim>::max_children_per_cell;
   this->dof_access_index =
     internal::MatrixFreeFunctions::DoFInfo::dof_access_cell;
 
+  constexpr unsigned int n_lanes = VectorizedArrayType::size();
```

Comment:
Is it performance relevant that this line is defined outside the if-else? I think the code would be more readable if defined inside the if/else.